src/helpers/paper-util.js

```diff
@@ -352,6 +439,45 @@ const PaperUtil = {
       func(inGridBoundsPoint, ...args)
     }
 
+    this.getTargetPointForKeyboardEvent = (event) => {
+      const focusedPath = this.getFocusedStartingPoint()
+      const ALLOWED_KEYS = ['up', 'down', 'left', 'right']
+      if(focusedPath !== null && focusedPath !== undefined && ALLOWED_KEYS.includes(event.key)) {
+        const gridPoint = this.fromViewCoordinateToGrid(focusedPath.position)
+        const targetPoint = (() => {
+          switch(event.key) {
```

Comment:
Are these `event.key` values stable? Are there any compatibility modes to consider?

Reply:
Yes, these are stable as per [paperjs](http://paperjs.org/reference/keyevent/#key)